awx_collection/tests/integration/targets/project/tasks/main.yml

```diff
@@ -195,6 +195,21 @@
         that:
           - result.changed
 
+    - name: Set project to remote archive and test that it updates correctly.
+      project:
+        name: "{{ project_name3 }}"
+        organization: Default
+        scm_type: archive
+        #  This should be updated to the ansible repo once a release is made.
+        scm_url: https://github.com/sean-m-sullivan/test-playbooks/archive/refs/tags/v1.0.0.tar.gz
```

Comment:
I made a release of test-playbooks for you to use here:
https://github.com/ansible/test-playbooks/archive/refs/tags/1.0.0.tar.gz

Once you get this updated I can test this and get this merged.

Reply:
@john-westcott-iv Updated to the release, thank you for that!